parsec/core/fs/workspacefs/file_transactions.py

```diff
@@ -249,7 +249,7 @@ async def fd_resize(self, fd: FileDescriptor, length: int, truncate_only: bool =
                 return
 
             # Perform the resize operation
-            await self._manifest_resize(manifest, length)
+            await self._manifest_resize(manifest, length, cache_only=True)
```

Comment:
Actually no, this is just to be consistent with the `fd_write` method which is also cache-only. This way, using `write_bytes` typically flushes the cache only once, when closing the file (instead of flushing once when truncating to zero then once more when closing).